File: src/main/java/jokatu/components/controllers/HomeController.java
package jokatu.components.controllers;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;

/**
 * @author Steven Weston
 */
@Controller
public class HomeController {

	/**
	 * Go to the login page.
	 */
	@RequestMapping("/login")
	String login() {
		return "views/log_in";
	}

	/**
	 * Go to the home page.
	 */
	@RequestMapping("/")
	String loggedIn() {
		return "views/home";
	}
}

File: src/main/kotlin/jokatu/components/controllers/HomeController.kt
package jokatu.components.controllers

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping

/**
 * @author Steven Weston
 */
@Controller
class HomeController {

	/**
	 * Go to the login page.
	 */
	@RequestMapping("/login")
	internal fun login(): String {
		return "views/log_in"
	}

	/**
	 * Go to the home page.
	 */
	@RequestMapping("/")
	internal fun loggedIn(): String {
		return "redirect:/game/0"
	}
}
